package.json

```diff
@@ -89,6 +89,7 @@
     ]
   },
   "dependencies": {
+    "@shopify/oxygen-cli": "file:../oxygen-cli",
```

Comment:
Package and package lock files will be reverted once oxygen-cli is updated. These changes will not be merged.